README.md

```diff
@@ -8,30 +8,30 @@ Setup
 
 You need Python 3.6+.
 
-My recommended setup process is:
+My recommended installation method is [pipx](https://pypi.org/project/pipx/). If you don't have it yet, [install pipx](https://pipxproject.github.io/pipx/installation/) with:
 
-    $ pyvenv venv
-    $ source venv/bin/activate
-    $ pip install -r requirements.txt
+    $ pip install pipx
 
-...adjust as needed. Just make sure the dependencies from `requirements.txt` get installed somehow.
+Then you can install leech with:
+
+    $ pipx install --spec git+https://github.com/kemayo/leech.git leech
```

Comment:
If you distribute to PyPI, this will become the much more streamlined

```
$ pipx install leech-webfic
```
(or whatever PyPI distribution name you end up with.)